redaxo/src/core/lib/rex.php

```diff
@@ -131,15 +131,35 @@ public static function setProperty($key, $value)
      * @throws InvalidArgumentException on invalid parameters
      *
      * @return mixed The value for $key or $default if $key cannot be found
+     * @psalm-return (
+     *     $key is 'login' ? rex_backend_login :
+     *     ($key is 'timer' ? rex_timer :
+     *     ($key is 'lang' ? string :
+     *     ($key is 'instname' ? string :
+     *     ($key is 'theme' ? string :
```

Comment:
```suggestion
     *     ($key is 'theme' ? string|null :
```

Im Auto-Modus ist es null.